test/functional/rpc_zmq.py

```diff
@@ -17,6 +17,11 @@ def set_test_params(self):
         self.setup_clean_chain = True
 
     def run_test(self):
+        try:
+            import zmq
```

Comment:
I believe it should see if zmq is compiled into bitcoind (see the test config file)